homeassistant/components/synology_dsm/camera.py

```diff
@@ -73,12 +73,11 @@ def __init__(
         Camera.__init__(self)
 
         self._camera_id = camera_id
-        self._api = api
 
     @property
-    def camera_data(self):
+    def camera_data(self) -> SynoCamera:
```

Comment:
In case `coordinator.data` is None.
```suggestion
    def camera_data(self) -> SynoCamera | None:
```

Reply:
If it's not possible that `data` doesn't have `"cameras"` key or that dict doesn't `self._camera_id`, then just add an assert. Otherwise return `None` if one of the keys is missing.